.github/workflows/build.yml

```diff
@@ -15,11 +15,7 @@ on:
 
 jobs:
   build:
-    runs-on: ubuntu-20.04
-    container:
-      image: 3mdeb/coreboot-sdk:mkimage
```

Comment:
@arturkow2000 Do we really need to that? To call the `docker run` for each command? With local gitlab-runner we simply use the same job syntax. Each time I use GA currently, I feel that  the gitlab CICD is so much better IMHO.

Reply:
@macpijan GA detects that runner is inside Docker and fails
with `Error: Container feature is not supported when runner is already running inside container.`
https://github.com/3mdeb/coreboot/runs/2930800018?check_suite_focus=true#step:2:1

Reply:
This looks terrible, but might be the only workaround for now.